src/components/NavBar/Navbar.tsx

```diff
@@ -1,28 +1,106 @@
 import Web3Status from 'components/Web3Status'
 import { useWindowSize } from 'hooks/useWindowSize'
+import { ReactNode } from 'react'
+import { NavLink, NavLinkProps, useLocation } from 'react-router-dom'
 
+import { Box } from '../../nft/components/Box'
+import { Row } from '../../nft/components/Flex'
+import { UniIcon, UniIconMobile } from '../../nft/components/icons'
 import { breakpoints } from '../../nft/css/sprinkles.css'
+import { MenuDropdown } from './MenuDropdown'
+import { MobileSideBar } from './MobileSidebar'
 import * as styles from './Navbar.css'
 
+interface NavLinkItemProps {
+  href: string
+  id?: NavLinkProps['id']
+  isActive?: boolean
+  children: ReactNode
+}
+
+const NavLinkItem = ({ href, id, isActive, children }: NavLinkItemProps) => {
+  return (
+    <NavLink
+      to={href}
+      className={isActive ? styles.activeMenuItem : styles.menuItem}
+      id={id}
+      style={{ textDecoration: 'none' }}
+    >
+      {children}
+    </NavLink>
+  )
+}
+
 const MobileNavbar = () => {
   return (
     <>
-      <nav className={styles.nav} />
-      <div className={styles.mobileWalletContainer}>
+      <nav className={styles.nav}>
+        <Box display="flex" height="full" flexWrap="nowrap" alignItems="stretch">
+          <Box className={styles.leftSideMobileContainer}>
+            <Box as="a" href="#/swap" className={styles.logoContainer}>
+              <UniIconMobile width="44" height="44" className={styles.logo} />
+            </Box>
+            {/* TODO add ChainSwitcher */}
+          </Box>
+          <Box className={styles.rightSideMobileContainer}>
+            <Row gap="16">
+              {/* TODO add Searchbar */}
+              <MobileSideBar />
+            </Row>
+          </Box>
+        </Box>
+      </nav>
+      <Box className={styles.mobileWalletContainer}>
         <Web3Status />
-      </div>
+      </Box>
     </>
   )
 }
 
 const Navbar = () => {
   const { width: windowWidth } = useWindowSize()
+  const { pathname } = useLocation()
```

Comment:
Move this to after the early return.

Reply:
Calling the hook conditionally breaks our eslint rules: `React Hook "useLocation" is called conditionally. React Hooks must be called in the exact same order in every component render.`

Reply:
👍 . Could you move the `isPoolActive` logic after the early return, as it's part of the subsequent logic?